src/dom/dom.js

```diff
@@ -377,29 +560,54 @@ p5.prototype.createSpan = function (html = '') {
 };
 
 /**
- * Creates an `&lt;img&gt;` element in the DOM with given src and
- * alternate text.
+ * Creates an `&lt;img&gt;` element.
```

Comment:
Maybe worth differentiating this from `p5.Image`: this exists *outside* of the canvas while `p5.Image` is only visible when you draw it on the canvas